recipes/mender-monitor/debian-master/mender-monitor.install

```diff
@@ -0,0 +1,27 @@
+/usr/bin/mender-monitord
+/usr/bin/mender-monitorctl
+
+/etc/mender-monitor/monitor.d/service.sh
+/etc/mender-monitor/monitor.d/log.sh
+/etc/mender-monitor/monitor.d/dbus.sh
+
+usr/share/mender-monitor/mender-monitord
+usr/share/mender-monitor/mender-monitorctl
+usr/share/mender-monitor/.version
+usr/share/mender-monitor/daemon.sh
+usr/share/mender-monitor/ctl.sh
+usr/share/mender-monitor/common/common.sh
+usr/share/mender-monitor/common/log.sh
+usr/share/mender-monitor/common/setup.sh
+usr/share/mender-monitor/lib/monitor-lib.sh
+usr/share/mender-monitor/lib/array-lib.sh
+usr/share/mender-monitor/lib/ctl-lib.sh
+usr/share/mender-monitor/lib/buf-lib.sh
+usr/share/mender-monitor/lib/fixlenstore-lib.sh
+usr/share/mender-monitor/lib/confpush-lib.sh
+usr/share/mender-monitor/lib/mender-client-lib.sh
+usr/share/mender-monitor/lib/log-lib.sh
+usr/share/mender-monitor/lib/alert-lib.sh
+usr/share/mender-monitor/lib/expiry-lib.sh
+usr/share/mender-monitor/lib/service-lib.sh
+usr/share/mender-monitor/config/config.sh
```

Comment:
Can we do all these a `.dirs` file? I am concern we will later add files in the repo and miss them here.

_EDIT_: This might be okey, see my comment below